### ISSUE TITLE
Undefined values vs undefined properties

### ISSUE THREAD

**Issue by thejsj** [tjmehta/keypather#18]
```
> a = {} 
> keypather.get(a, 'PROPERTY.THAT.DOESNT.EXIST')
null
> keypather.get(a, 'PROPERTY_THAT_DOESN_EXIST')
undefined
```

This seems like weird behavior to me. I would have expected either both to be `null` or both to be `undefined`.

I understand that it's probably a bit too late to change this, but thought I'd point it out. It seems to me that a value that doesn't exist should always return `undefined` since it's... well... undefined, but I might be missing something.

**Comment by thejsj** [comment]
Now I have to do this: 

```
        var value = keypather.get(model, key)
        if (value !== undefined && value !== null) {
          // ...
        }
```

**Comment by tjmehta** [comment]
Interesting will check this out probably Sunday. For now you can use != null that will cover both null and undefined

**Comment by tjmehta** [comment]
Could just use 101/exists as well

**Comment by tjmehta** [comment]
fixed:: keypather@v2.0.0
